tools/walletextension/wallet_extension.go

```diff
@@ -118,9 +123,9 @@ func (we *WalletExtension) handleHTTPEthJSON(resp http.ResponseWriter, req *http
 	//}
 
 	// We forward the request on to the Obscuro node.
-	nodeResp, err := forwardMsgOverWebsocket(websocketProtocol+we.nodeAddr, body)
+	nodeResp, err := forwardMsgOverWebsocket(websocketProtocol+we.hostAddr, body)
```

Comment:
Should this be handled by the obscuro client? Feels odd to have a client and then need to know the address and communicate with it separately.

Reply:
Yeah, you're right, maybe in the future.